packages/kbn-ui-shared-deps/webpack.config.js

```diff
@@ -105,6 +108,28 @@ exports.getWebpackConfig = ({ dev = false } = {}) => ({
   },
 
   optimization: {
+    minimizer: [
+      new CssMinimizerPlugin({
+        minimizerOptions: {
+          preset: [
+            'default',
+            {
+              discardComments: false,
+            },
+          ],
+        },
+      }),
+      new TerserPlugin({
```

Comment:
What affect did Terser have?

Reply:
No effect - it's used by default but by adding a list of minimizers we had to define it.